### ISSUE TITLE
createPatch -> applyPatch failed.

### ISSUE THREAD

**Issue by mog422** [kpdecker/jsdiff#57]
```
var diff = require('./diff');

var oldtext = 'AAAAAAAAAAAAAAAA\n\n';
var newtext = 'AAAAAAAAAAAAAAAA\nBBBBBB' + String.fromCharCode(8232) + '\nCCCCCCCCCCCCCCCCCC\n\n';


var diffed = diff.createPatch('test', oldtext, newtext);
console.log(diff.applyPatch(oldtext, diffed) !== newtext ? 'failed' : 'success');
```
This code result must be success.
But It's failed.

**Comment by kpdecker** [comment]
Thanks for filing the issue. It looks like we are not handling the various unicode line control characters properly. I've updated it so only ASCII new lines are considered for patch creation.

**Comment by kpdecker** [comment]
Released in 1.4.0
